string TYPE=drone

uint8 value
